honeybee_radiance_command/options/dctimestep.py

```diff
@@ -0,0 +1,87 @@
+"""Dctimestep parameters"""
+from .optionbase import OptionCollection, StringOption, IntegerOption, BoolOption, \
+    StringOptionJoined
+
+
+class DctimestepOptions(OptionCollection):
+    """
+    [-n nsteps] [-h][-o ospec][-i{f|d}][-o{f|d|c}]
+
+    Also see: https://www.radiance-online.org/learning/documentation/manual-pages/pdfs/dctimestep.pdf
+    """
+
+    __slots__ = ('_n', '_h', '_o', '_i', '_op_fmt')
+
+    def __init__(self):
+        """dctimestep command options."""
+        OptionCollection.__init__(self)
+        self._on_setattr_check = False
+        self._h = BoolOption('h', 'remove header in output file. default:header is '
+                                  'written')
+        self._i = StringOptionJoined('i', 'input data format for the sky vector. "f" '
+                                          'indicates float and "d" indicates double. '
+                                          'Double is efficient for large matrices. This'
+                                          'option is unnecessary if the sky vector/matrix '
+                                          'includes a header', valid_values=['d', 'f'],
+                                     whole=False)
+        self._o = StringOption('o', 'The -o option may be used to specify a file or '
+                                    'a set of output files to use rather than the '
+                                    'standard output. If the given specification '
+                                    'contains a "%d" format string, this will be '
+                                    'replaced by the time step index, starting from 0.'
+                                    'In this way, multiple output pictures or '
+                                    'separate result vectors may be produced.')
+        self._op_fmt = StringOptionJoined('o', 'The -of, -od or -oc option may be used to'
```

Comment:
I thought you want to use capital O in this case?

Reply:
Yeah, the o versus O thing was turning out to be confusing. I'd rather that it looks ugly (but clear) than pretty (but confusing). Anyway, the first variable ensures that the input is set as '-o' as far as talking to Radiance goes, so I think we are all good.